### PR TITLE
[REST] unify rules to save a group

### PR DESCRIPTION
**Pull request by ndufreche**
An unification of the saving rules on group
see => backbee/BbCoreJs#694

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
This need a new test in Rest/Tests/PageControllerTest.

**Comment by eric-chau** [comment]
Don't merge it until we add the test, @mickaelandrieu is right. If you don't have the time to do it I will do it. Thanks for your contribution @ndufreche.

**Comment by ndufreche** [comment]
I will take a little time tomorrow to complete the PR and it's actually broken.

File: Rest/Controller/GroupController.php [hunk 1]
```diff
@@ -98,6 +98,16 @@ public function deleteAction(Group $group)
      */
     public function putAction(Group $group, Request $request)
     {
+        $site = $this->getSite($request);
+
+        if ($this->isDuplicated($request->request->get('name'), $site)) {
+            return new JsonResponse([
```
Comment (mickaelandrieu) [inline Rest/Controller/GroupController.php]: you need to add JsonResponse use statment

Comment (ndufreche) [inline Rest/Controller/GroupController.php]: Yeah it was corrected but not commited :)

Comment (mickaelandrieu) [inline Rest/Controller/GroupController.php]: Nice :bear:

**Comment by mickaelandrieu** [comment]
Ok for me, if build is green I merge /c @eric-chau

**Pull request merged**